templates/CMakeLists.txt.template

```diff
@@ -170,6 +170,7 @@
   option(gRPC_BUILD_CODEGEN "Build codegen" ON)
   option(gRPC_BUILD_CSHARP_EXT "Build C# extensions" ON)
   option(gRPC_BACKWARDS_COMPATIBILITY_MODE "Build libraries that are binary compatible across a larger number of OS and libc versions" OFF)
+  option(gRPC_INSTALL_PLUGINS "Install protobuf plugins (disable e.g. when building for a pure cross-compilation environment" ON)
```

Comment:
> Btw, in this particular case, would just setting gRPC_BUILD_GRPC_*_PLUGIN to OFF solve the issue?

Technically yes, that's like a shortcut for switching off all the plugins. Perhaps, following the semantics, it would be better to use the name `gRPC_BUILD_PLUGINS` then? I'd say having one option switching off all the plugins is better than switching them off one by one, isn't it?

Reply:
Why isn't just setting `gRPC_BUILD_CODEGEN=OFF` not sufficient?

Reply:
I'll give a try again with my use cases and try to provide an update later this week

Reply:
It seems `gRPC_BUILD_CODEGEN=OFF` also switches off `grpc++_reflection` target, which is not a plugin.

Reply:
@jtattermusch Current yocto recipe in meta-openembedded uses only CPP plugin, so my use case should be covered if I simply switch it off for cross-compilation build. Bottom line for this issue, my previous commit can be reverted https://github.com/grpc/grpc/pull/26148.